transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -209,12 +221,13 @@ private void registerService(BundleContext bundleContext) {
 
     private List<String> getSupportedMimeTypes() {
         SortedSet<MediaType> mediaTypes = MediaTypeRegistry.getDefaultRegistry().getTypes();
-        List<String> mimeTypes = new ArrayList<String>(mediaTypes.size());
+        List<String> mimeTypes = new ArrayList<>(mediaTypes.size());
 
         for (MediaType mediaType : mediaTypes) {
             String mimeType = mediaType.getType() + "/" + mediaType.getSubtype();
             mimeTypes.add(mimeType);
         }
+        mimeTypes.add("image/jp2");
```

Comment:
We are going to upgrade to 1.7 in the near future, so if they do have support for jp2 then it may be something to look at.